templates/layout.html

```diff
@@ -47,18 +47,16 @@
   </div>
   <div class="navbar-menu" id="navMenu">
     <div class="navbar-start">
-       <a {% if this._path == '/' %} class="is-active navbar-item"{% else %}class="navbar-item"{% endif %} href="{{ '/'|url }}">w3lc0m3!</a>
```

Comment:
You can't come back to home "/" without this link. Suggest keeping this, but with an other name.

Reply:
You can always click on the icon (which is imho known to most internet users)

Reply:
lgtm

Reply:
for me the name ("w3lc0me!") is a reason for keeping it; but it's simply a copy of the logo; therefore suggesting to remove it - did we overlook something?

Reply:
The leetspeak can also be found on the landing page 😋

Reply:
true! :)